src/OutGridView.Gui/ViewModels/FilterQueryBuilderViewModel.cs

```diff
@@ -37,7 +37,7 @@ public class FilterQueryBuilderViewModel : ViewModelBase
         public Boolean IsColumnSelectVisible { [ObservableAsProperty] get; }
 
         //Placeholder hack for combo box
-        private DataTableColumn placeholderColumn = new DataTableColumn("Add Column Filter", -1, 1.GetType().FullName, "");
+        private DataTableColumn placeholderColumn = new DataTableColumn("Add Column Filter", 1.GetType().FullName);
```

Comment:
```suggestion
        private DataTableColumn placeholderColumn = new DataTableColumn("Add Column Filter", 1.GetType().FullName);
```

instead of `1.GetType()` can you just put the FullName in here since it's static